### PR TITLE
delete fontface polyfill

### PR DESCRIPTION
**Pull request by alanwutang11**
support or IE11 and old Edge has been removed. Removed FontFace Polyfills as well 

fixes: https://github.com/flutter/flutter/issues/115013

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide] and the [C++, Objective-C, Java style guides].
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt. See [testing the engine] for instructions on writing and running engine tests.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I signed the [CLA].
- [x] All existing and new tests are passing.

### CODE REVIEW

**Review by hterkelsen — CHANGES_REQUESTED** [review]
LGTM if Yegor's suggested changes are made

File: lib/web_ui/lib/src/engine/text/font_collection.dart [hunk 1]
```diff
@@ -106,23 +102,14 @@ class HtmlFontCollection implements FontCollection {
...
-  factory FontManager() {
-    if (supportsFontLoadingApi) {
-      return FontManager._();
-    } else {
-      return _PolyfillFontManager();
-    }
-  }
 
-  FontManager._();
+  FontManager();
```
Comment (hterkelsen) [inline lib/web_ui/lib/src/engine/text/font_collection.dart:112]: you don't need to declare an empty constructor

**Review by hterkelsen — APPROVED** [review]
LGTM with nit

**Comment by Hixie** [comment]
test-exempt: removing code

**Pull request merged**